src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -207,20 +207,27 @@
 (defn text-area [field-descriptor & {:keys [div-kwd] :or {div-kwd :div.application__form-field}}]
   (let [application (subscribe [:state-query [:application]])
         answers     (subscribe [:state-query [:application :answers]])
-        on-change   (partial textual-field-change field-descriptor)]
+        on-change   (partial textual-field-change field-descriptor)
+        size        (-> field-descriptor :params :size)
+        max-length  (-> field-descriptor :params :max-length (or 1000))]
```

Comment:
Is 1000 characters a sane default max-length? Seems a bit low?

Reply:
This was just a preliminary suggestion for default values of S/M/L boxes with 500/1000/1500 chars respectively. We'll adjust if needed.